bolt/lib/Profile/StaleProfileMatching.cpp

```diff
@@ -51,6 +51,12 @@ cl::opt<bool>
                       cl::desc("Infer counts from stale profile data."),
                       cl::init(false), cl::Hidden, cl::cat(BoltOptCategory));
 
+cl::opt<unsigned> StaleMatchingMinMatchedBlock(
+    "stale-matching-min-matched-block",
+    cl::desc("Percentage threshold of matched basic blocks at which stale "
+             "profile inference is executed."),
+    cl::init(50), cl::Hidden, cl::cat(BoltOptCategory));
```

Comment:
Do we want to leave the default same as current behavior for now?

Reply:
Yes, changed the default to 0